src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return this.color + " " + this.number;
```

Comment:
imagine that we have 50 more fields that describe a ball, it would be quite challenging to understand what is what if we describe it like that, so we need to add an indication of what field has what value, and make it JSON formatted to make it even prettier, so I suggest using the default format that we have in code generation feature of the IDE:
```suggestion
        return "Ball{"
                + "color=" + color
                + ", number=" + number
                + '}';
```